### PR TITLE
RHMAP-18944 macos-setup review

### PR DESCRIPTION
**Pull request by pmacko1**
splinter from https://github.com/aerogear/aerogear.org/pull/683/files 
https://issues.jboss.org/browse/RHMAP-18944

### CODE REVIEW

File: docs/digger/installation/macos-setup.asciidoc [hunk 1]
```diff
@@ -1,46 +1,44 @@
-=== Setup macOS Servers
+[[macos-setup]]
```
Comment (laurafitzgerald) [inline docs/digger/installation/macos-setup.asciidoc]: This file is not included in the installation guide. Should it be?

Comment (pmacko1) [reply docs/digger/installation/macos-setup.asciidoc]: it is https://github.com/aerogear/aerogear.org/pull/712/files

File: docs/digger/installation/macos-setup.asciidoc [hunk 2]
```diff
@@ -1,46 +1,44 @@
...
+After setting the required Ansible variables, the installer will perform the following tasks to provision macOS servers:
 
-. Install Homebrew.
-. Install RVM and Ruby.
-. Install NVM and Node.
-. Install Xcode.
-. Download Apple certificates.
-. Update Cocoapods.
-. Configure machine as a node in Jenkins.
+. Install Homebrew,
```
Comment (laurafitzgerald) [inline docs/digger/installation/macos-setup.asciidoc]: @pmacko1 not sure why we are using any punctuation here. I think with points there is no need for punctuation right?

File: docs/digger/installation/macos-setup.asciidoc [hunk 3]
```diff
@@ -55,5 +53,5 @@ ansible-playbook -i sample-inventory-file sample-playbook.yml
 
 ==== Further macOS Configuration options
 
-Further configuration of the macOS node is possible through optional variables.
-link:macos-options[View all macOS variables].
\ No newline at end of file
+xref:macos-variables[View all macOS variables].
```
Comment (laurafitzgerald) [inline docs/digger/installation/macos-setup.asciidoc]: I think these are better the other way around. wdyt? make the comment and then provide the link?

File: docs/digger/installation/macos-setup.asciidoc (hunk 1 above)
Comment (laurafitzgerald) [reply docs/digger/installation/macos-setup.asciidoc]: @pmacko1 thanks, that resolves the above query

File: docs/digger/installation/macos-setup.asciidoc (hunk 2 above)
Comment (pmacko1) [reply docs/digger/installation/macos-setup.asciidoc]: It depends. Here, I think you are correct.

File: docs/digger/installation/macos-setup.asciidoc (hunk 1 above)
Comment (pmacko1) [reply docs/digger/installation/macos-setup.asciidoc]: resolved

File: docs/digger/installation/macos-setup.asciidoc (hunk 2 above)
Comment (pmacko1) [reply docs/digger/installation/macos-setup.asciidoc]: resolved

File: docs/digger/installation/macos-setup.asciidoc (hunk 3 above)
Comment (pmacko1) [reply docs/digger/installation/macos-setup.asciidoc]: resolved

**Comment by pmacko1** [comment]
@laurafitzgerald i've just resolved your comments, polished the text a little, ok to merge?

**Comment by laurafitzgerald** [comment]
@pmacko1 thanks for the updates. Feel free to merge this one.

**Pull request merged**